cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/metrics/MetricsCollector.java

```diff
@@ -166,8 +160,13 @@ private Measurement createMeasurement(String service, double value) {
             final List<MBeanAttributeInfo> filteredAttributes = new ArrayList<>();
 
             for (MBeanAttributeInfo attributeInfo : attributes) {
-                if (allowedTypes.contains(attributeInfo.getType())) {
+                try {
+                    final Object obj = mbeanServerConn
+                            .getAttribute(objectInstance.getObjectName(), attributeInfo.getName());
+                    Double.parseDouble(obj.toString());
```

Comment:
I do not get this, `parseDouble` is here only to throw exception and escape from try block?

Can you add a comment here since I needed to read this 5 times to figure out what is going on. Also in catch block since now it is just eating up exception and comment it so that we knoe it is expected.

Or even better can you try with `if (obj instanceof Number)` instead of this ugly try/catch. All numeric types are subclasses of `Number` so this should work and give you exactly what you want.